tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Web/Controllers/MatterController.cs

```diff
@@ -904,30 +904,7 @@ public IActionResult AssignContentType([FromBody] MatterMetadata matterMetadata)
             // For each value in the list of Content Type Names
             // Add that content Type to the Library
             Matter matter = matterMetadata.Matter;
-            Client client = matterMetadata.Client;

```

Comment:
This code is not necessary as we are getting correct json request object from the client